homeassistant/components/whirlpool/climate.py

```diff
@@ -71,10 +68,10 @@ async def async_setup_entry(
     """Set up entry."""
     appliances_manager = config_entry.runtime_data
     aircons = [AirConEntity(hass, aircon) for aircon in appliances_manager.aircons]
-    async_add_entities(aircons, True)
+    async_add_entities(aircons)
```

Comment:
no need to update before add, since data is fetched when the integration is setup